src/middlewared/middlewared/plugins/virt/instance.py

```diff
@@ -144,12 +180,19 @@ def __data_to_config(self, data: dict, raw: dict = None):
 
         if data.get('autostart') is not None:
             config['boot.autostart'] = str(data['autostart']).lower()
+
+        if instance_type == 'VM':
+            if data.get('enable_vnc') and data.get('vnc_port'):
+                config['raw.qemu'] = f'-vnc :{data["vnc_port"] - VNC_BASE_PORT}'
+            if data.get('enable_vnc') is False:
+                config['raw.qemu'] = ''
```

Comment:
These will wipe out any other current raw.qemu value. Wonder if we can be more friendly here and perhaps append to current value instead. At very least perhaps store the old existing value in a user config. Any thoughts?